### ISSUE TITLE
LCD_I2C_PANELOLU2 not working correctly with latest RC

### ISSUE THREAD

**Issue by ewookscorp** [MarlinFirmware/Marlin#4030]
I compiled latest source code ( 1.1.0-RC6 )  for my prusa i3 clone (sanguinololu 1.3 + ic2 lcd + sdcard)
but lcd panel just blinking and beebing, menu rolling itself and changes views automatically like "select" is pushed down constantly???

All are working fine with old stabile release

**Comment by thinkyhead** [comment]
Does this problem still exist in the `RCBugFix` branch? We've done a lot of patches since `RC6` and might have fixed this one. If the problem still exists in the latest code, then we will have a good starting point to find and fix the issue.

**Comment by ewookscorp** [comment]
Hmm I took now that RCBugFix branc but now when I trying to compile it I got loooot's of errors.
I will look more this.  Compile error happens when  LCD_I2C_PANELOLU2 is defined with out it there is no errors.

Short sample bellow and full log found here http://pastebin.com/eq46gaZR

```
 C:\Users\sami\AppData\Local\Temp\build15c6831deedbd5d6201e66142622daa3.tmp\sketch\Configuration.h:1200,

                 from C:\projektit\Marlin-RCBugFix\Marlin-RCBugFix\Marlin\Marlin.ino:37:

fastio.h:47: error: pasting "DIO" and "-" does not give a valid preprocessing token

 #define _READ(IO) ((bool)(DIO ## IO ## _RPORT & MASK(DIO ## IO ## _PIN)))

                           ^
```

**Comment by Blue-Marlin** [comment]
@thinkyhead 
This could be a side effect of the MAX6675 fix. We looked after the sc-cards but not for the displays.

**Comment by fabtopia** [comment]
i have the exact same configurations and use Rc6 with no display problems. can't there be a loose connector

**Comment by ewookscorp** [comment]
Same HW works fine with older stabile version, so I think connectors are ok.

**Comment by fabtopia** [comment]
Is it right from switching it on or does it starts after doing something?

**Comment by ewookscorp** [comment]
The problem begins when a new firmware (RC6) starts. I will be double check my config and try again to compile RC6 since that RCBugFix branc not compiled.

**Comment by thinkyhead** [comment]
I would test-compile this, but don't have `LiquidTWI2.h` installed at the moment. I'll have to continue tomorrow.

**Comment by ewookscorp** [comment]
Ok I now pulled RC6 version via GIT intead download ZIP file. 
I checked my configuration compiled firmware and uploded it in my device.
Display seems works now with fresh source.
Since I can now continue to config my device this case can be closed. This was possible my own mistake in code so sorry about this. Thanks to help for every body and keep good job continue :)

X homing does seems not work with this firmware, weird? need to be look this now.
